editor/editor_help.cpp

```diff
@@ -368,8 +368,30 @@ void EditorHelp::_add_method(const DocData::MethodDoc &p_method, bool p_overview
 	class_desc->pop();
 	if (!p_method.qualifiers.is_empty()) {
 		class_desc->push_color(qualifier_color);
-		class_desc->add_text(" ");
-		_add_text(p_method.qualifiers);
+
+		PackedStringArray qualifiers = p_method.qualifiers.split_spaces();
+		for (int i = 0; i < qualifiers.size(); i++) {
+			const String qualifier = qualifiers[i];
```

Comment:
```suggestion
		for (const String &qualifier : qualifiers) {
```

Reply:
Hey, I can keep that snippet of code in mind for the future.